includes/Block/Web_Stories_Block.php

```diff
@@ -182,6 +182,7 @@ private function get_script_settings() {
 			'config'     => [
 				'maxNumOfStories' => $this->max_num_of_stories,
 				'editStoryURL'    => $edit_story_url,
+				'archiveURL'      => get_post_type_archive_link( $rest_base ),
```

Comment:
$rest_base != $post_type. 

This variable is confusing. 

You should have the rest base to look more like [this](https://github.com/google/web-stories-wp/blob/31dffad61c908082a721218f428b7a88ea0ea285/includes/Story_Post_Type.php#L505).